pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -44,8 +43,7 @@ std::unordered_set<std::string> GetTorchMlirWhiteList() {
       "aten::mul",      "aten::eq",          "aten::lt",
       "aten::ne",       "aten::__and__",     "aten::floor_divide",
       "prim::Constant", "aten::expand_as",   "aten::repeat",
-      "aten::expand",
-  };
+      "aten::expand",   "aten::hardtanh"};
```

Comment:
Should we sort this op list by name (e.g. one line on op, disable clang-format here)？

Reply:
agree with you, will do that.

Reply:
this enhancement has implemented in another PR.